src/sage/geometry/cone_critical_angles.py

```diff
@@ -0,0 +1,998 @@
+r"""
+Find maximal angles between polyhedral convex cones
+
+Finding the maximal (or equivalently, the minimal) angle between two
+polyhedral convex cones is a hard nonconvex optimization problem. The
+problem for a single cone was introduced in [IS2005]_, and was later
+extended in [SS2016]_ to two cones as a generalization of the
+principal angle between two vector subspaces.
+
+Seeger and Sossa proposed an algorithm to find maximal angles, and
```

Comment:
```suggestion
Seeger and Sossa proposed an algorithm in [SS2016]_ to find maximal angles, and
```
Good to be explicit IMO.

Reply:
No problem, done